mlrun/config.py

```diff
@@ -346,6 +346,7 @@
         },
         "batch_processing_function_branch": "master",
         "parquet_batching_max_events": 10000,
+        "store_type": "kv",
     },
```

Comment:
add comment pointing to the class with the model endpoint store type options

Reply:
fixed
```
        # See mlrun.api.schemas.ModelEndpointStoreType for available options 
        "store_type": "kv",
```